src/components/SiteAlert/_site-alert.scss

```diff
@@ -93,3 +93,8 @@
     width: calc(1.5 * var(--space-unit));
   }
 }
+
+.site-alert__text {
+  width: 100%;
```

Comment:
Looks like there is already a rule in this file for `.site-alert__text`

Reply:
Ah, missed that. (Something I thought the linter would pick up... will look at that separately)